language/move-lang/functional-tests/tests/libra/epilogue/cant_call_epilogue_in_transaction.move

```diff
@@ -0,0 +1,9 @@
+script {
+use 0x1::LibraAccount;
+use 0x1::Coin1::Coin1;
+
+fun main(account: &signer) {
+    LibraAccount::epilogue<Coin1>(account, 0, 0, 0, 0);
+}
+}
+// check: MoveSourceCompilerError internal to module
```

Comment:
This test will be removed, the same thing is tested in `move_check/typing/module_call_internal.move`.